scene/gui/split_container.cpp

```diff
@@ -196,6 +201,19 @@ Control *SplitContainer::_get_sortable_child(int p_idx, SortableVisibilityMode p
 	return nullptr;
 }
 
+void SplitContainer::_fit_child_in_rect_with_visibility_update(Control *p_child, const Rect2 &p_rect) {
+	// For very small rects (like when a size is set to 0 via autosnapping) hide the child instead
```

Comment:
This is misleading, because visibility is not being changed.

Reply:
Depends on how you think about it, from a technical perspective it changes the scale so it doesn't actually change its visibility flag, but from the user's perspective it makes the child disappear, so it does change its visibility. The goal of the function is to update the visibility from the end user's perspective even if from the technical side it changes its scale - hence the reason i added the comment to clarify what is going on and why.